lib/phase/permutation.js

```diff
@@ -22,16 +22,21 @@ module.exports = validatedDoc => (
                 annotations: Hoek.reach(doc, `jobs.${jobName}.annotations`, {
                     default: {}
                 }),
-                image: Hoek.reach(doc, `jobs.${jobName}.image`),
                 commands: Hoek.reach(doc, `jobs.${jobName}.commands`),
-                secrets: Hoek.reach(doc, `jobs.${jobName}.secrets`),
                 environment: Hoek.reach(doc, `jobs.${jobName}.environment`, {
                     default: {}
                 }),
+                image: Hoek.reach(doc, `jobs.${jobName}.image`),
+                secrets: Hoek.reach(doc, `jobs.${jobName}.secrets`),
                 settings: Hoek.reach(doc, `jobs.${jobName}.settings`, {
                     default: {}
                 })
             };
+
+            if ('description' in doc.jobs[jobName]) {
```

Comment:
There is a difference between this and `if (doc.jobs[jobName].description)`
Does it make sense to set the description to `''`, or should we just not set it?